### PR TITLE
Add Conv utility functions

### PR DESCRIPTION
**Pull request by magi345**
This PR introduces a new package that provides utility functions for converting golang data types with high performance. These functions pursuit maximum performance for building data-intensive applications. See README and tests for more details.

### CODE REVIEW

File: conv/const.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	{'6', '0'}, {'6', '1'}, {'6', '2'}, {'6', '3'}, {'6', '4'}, {'6', '5'}, {'6', '6'}, {'6', '7'}, {'6', '8'}, {'6', '9'},
+	{'7', '0'}, {'7', '1'}, {'7', '2'}, {'7', '3'}, {'7', '4'}, {'7', '5'}, {'7', '6'}, {'7', '7'}, {'7', '8'}, {'7', '9'},
+	{'8', '0'}, {'8', '1'}, {'8', '2'}, {'8', '3'}, {'8', '4'}, {'8', '5'}, {'8', '6'}, {'8', '7'}, {'8', '8'}, {'8', '9'},
+	{'9', '0'}, {'9', '1'}, {'9', '2'}, {'9', '3'}, {'9', '4'}, {'9', '5'}, {'9', '6'}, {'9', '7'}, {'9', '8'}, {'9', '9'},
+}
+
+// NaNb is byte slice for 'NaN'
+var NaNb = []byte{'N', 'a', 'N'}
+
+// NaN is byte slice for 'NaN'
```
Comment (yangru) [inline conv/const.go]: change the comment

**Comment by avamingli** [comment]
useful tools

File: conv/uint.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+		buf[pos+1], buf[pos] = digits2[u][1], digits2[u][0]
+	}
+
+	return buf[pos:]
+}
+
+func ui32Dig(u uint32, buf *[10]byte) []byte {
+	if u < 10 {
+		return digits1[u]
+	} else if u < 100 {
```
Comment (yangru) [inline conv/uint.go:76]: else if can be if directly

Comment (magi345) [reply conv/uint.go:76]: Ru and I confirmed the assemble code of `if ... if` and `if ... else`, there is no difference.

**Pull request merged**